schema/bom-1.7.proto

```diff
@@ -506,7 +506,7 @@ message Metadata {
   // The organization that supplied the component that the BOM describes. The supplier may often be the manufacture, but may also be a distributor or repackager.
   optional OrganizationalEntity supplier = 6;
   // The license information for the BOM document. This may be different from the license(s) of the component(s) that the BOM describes.
-  // EITHER (list of SPDX licenses and/or named licenses) OR (tuple of one SPDX License Expression)
+  // A list of SPDX licenses and/or named licenses and/or SPDX License Expression.
```

Comment:
Proposal 
> EITHER (list of SPDX licenses and/or named licenses) OR (list of SPDX License Expression)

this in combination with https://github.com/CycloneDX/specification/issues/619